docs/topics/proc_debug-logging-nodejs.adoc

```diff
@@ -0,0 +1,160 @@
+
+[discrete]
+=== Procedure
+
+. Start your application with the `DEBUG` environment variable set to enable debug logging.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ DEBUG=mybooster npm start
+----
++
+NOTE: The `DEBUG` environment variable uses link:https://www.npmjs.com/package/debug#wildcards[wildcards] to filter debugging messages.
+
+. Test your application to invoke debug logging. 
++
+For example, if you had debug logging in the xref:mission-http-api-nodejs[{name-mission-http-api} booster] to log the `name` variable in the `/api/greeting` method:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ curl http://localhost:8080/api/greeting?name=Sarah
+----
+
+. View your application logs to see your debug messages.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+mybooster name: Sarah +3m
+----
+
+
+== Show Debug Logging on OpenShift
+
+Use the the `DEBUG` environment variable in your application pod in OpenShift to enable debug logging.
+
+[discrete]
+=== Prerequisites 
+* An application with debug logging.
+* The `oc` CLI client installed.
+
+
+[discrete]
+=== Procedure
+
+. Use the `oc` CLI client to log into your OpenShift instance.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc login ...
+----
+
+. Deploy your application to OpenShift.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ npm run openshift
+----
+
+. Find the name of your pod and follow the logs to watch it start.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get pods
+....
+$ oc logs -f pod/POD_NAME
+----
++
+IMPORTANT: After your pod has started, keep following the logs.
+
+. Test your application. 
++
+For example, if you had debug logging in the xref:mission-http-api-nodejs[{name-mission-http-api} booster] to log the `name` variable in the `/api/greeting` method:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get routes
+...
+$ curl $APPLICATION_ROUTE/api/greeting?name=Sarah
+----
+
+. Return to your pod logs and notice there is no debug logging messages in the logs. 
+
+. Set the `DEBUG` environment variable in your application pod to enable debug logging.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc set env pod/POD_NAME DEBUG=mybooster
+----
+
+. Return to your pod logs to watch update roll out.
++
+IMPORTANT: After your update has rolled out, keep following the logs. This is where your debug logging will appear.
+
+. Test your application to invoke debug logging. 
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get routes
+...
+$ curl $APPLICATION_ROUTE/api/greeting?name=Sarah
+----
+
+. View your application logs to see your debug messages.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+...
+mybooster name: Sarah +3m
+----
+
+
+
+To disable debug logging, remove the `DEBUG` environment variable from the pod:
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc set env pod/POD_NAME DEBUG-
```

Comment:
Is this supposed to be `pod/POD_NAME DEBUG=`? Or does the `-` remove the env var?

Reply:
Adding a `-` at the end of an ENV VAR removes it as per:
https://docs.openshift.com/online/dev_guide/environment_variables.html#unset-environment-variables

Reply:
Thanks - I didn't know that. The more you know... ⭐️